.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Pull Request Checklist
+
+#### [] Description
+
+- [] Brief description of the changes made in this PR.
+
+#### [] Issue
+
+- **Fixes** #[issue_number]
```

Comment:
Can you remove the square brackets around the issue number which developers might forget to remove?